.github/workflows/serial.yml

```diff
@@ -17,3 +17,22 @@ jobs:
         run: make install
       - name: Test serial
         run: make test
+  cmake-build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install deps
+        run: sudo apt-get -y install gfortran cmake
+      - name: Configure serial
+        run: |
```

Comment:
What does a single pipe do here?

Reply:
I think it's used as a line continuation.